src/Graph.ts

```diff
@@ -222,7 +222,6 @@ export default class Graph {
 			for (const module of this.moduleById.values()) {
 				if (module instanceof Module) {
 					this.modules.push(module);
-					this.watchFiles[module.id] = true;
```

Comment:
Instead of adding files only once loading and parsing was successful, I think it is better to add them right away before loading so that the same logic can be used for the successful and the error case.